.github/workflows/ci.yml

```diff
@@ -631,6 +631,15 @@ jobs:
             docs-inventory-
       - name: "Build docs"
         run: breeze build-docs ${{ needs.build-info.outputs.docs-filter-list-as-string }}
+      - name: "Clone airflow-site"
+        run: >
+          git clone https://github.com/apache/airflow-site.git /workspace/airflow-site &&
```

Comment:
```suggestion
          git clone https://github.com/apache/airflow-site.git /workspace/airflow-site &&
```

This might be not the best place to check it out. I think You should use GITHUB_WORKSPACE as env variable

https://docs.github.com/en/actions/learn-github-actions/variables

Reply:
Thanks for bringing that up. Let me update it